.pipelines/singletenancy/cilium-overlay/cilium-overlay-e2e-step-template.yaml

```diff
@@ -152,6 +153,20 @@ steps:
     name: "ciliumConnectivityTests"
     displayName: "Run Cilium Connectivity Tests"
 
+  - script: |
+      echo "Run Hubble Connectivity Tests"
+      echo "update Cilium configmap to enable Hubble"
+      kubectl apply -f test/integration/manifests/cilium/cilium-config-hubble.yaml
+      kubectl rollout restart ds cilium -n kube-system
+      echo "wait 3 minutes for pods to be ready after restart"
+      sleep 180s
```

Comment:
What pods need to be ready?

Reply:
the Cilium pods, which are being restarted on L160

in lieu of static sleep, it could be a `kubectl wait`, ex: 
`kubectl wait --for=condition=ready pod -l k8s-app=cilium --timeout=3m`

imo this suite could benefit by using `kubectl wait` in several places instead of sleeps, including 
`kubectl wait --for=delete pod/ciliumidentity/etc`, but that's out of scope for this pr :)

Reply:
Would `kubectl rollout status ds -n kube-system cilium` be just as effective for this then?